notebooks/infer_predictions.ipynb

```diff
@@ -0,0 +1,180 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Run inference on pre-trained model:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import torch"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from grace.io.image_dataset import ImageGraphDataset\n",
+    "from grace.models.feature_extractor import FeatureExtractor\n",
+    "from grace.evaluation.visualisation import plot_simple_graph\n",
+    "from grace.evaluation.inference import GraphLabelPredictor\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Read some real grace-annotated data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "extractor_filename = \"/Users/kulicna/Desktop/classifier/extractor/resnet152.pt\"\n",
+    "pre_trained_resnet = torch.load(extractor_filename)\n",
+    "feature_extractor = FeatureExtractor(model=pre_trained_resnet)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "grace_path = \"/Users/kulicna/Desktop/dataset/shape_stars/train\"\n",
```

Comment:
In general we should remove path to personal directories if we want these notebooks to be run easily for everyone, we could add a new variable called: `input_paht = /path/to/data/` to be replaced by the user. 

Also, we need to put this data somewhere and easy to download.

Reply:
Fully agreed, we had this problem in the `examples\` folder where I replaced the path for a user-specified input: 
https://github.com/alan-turing-institute/grace/blob/488f60bf6822106129c1de1c83b244c051dfc827/examples/show_data.py#L13-L20
I'm not sure how to do this from the notebook tho...